doc/source/support/frequently-asked-questions.rst

```diff
@@ -144,6 +144,28 @@ How can I change who is authorized to release a package?
 
 In ``.github/workflows/build-wheel-release-upload.yml``, modify ``maintainer_github_username`` to the desired GitHub username. This username will be able to authorize the release by pushing the tag as instructed in :ref:`release-pypi-github`.
 
+.. _faq-release-ci-failed:
+
+Release CI failed. What should I do?
```

Comment:
Organization

pre-release
- authority problem
- pypi problem
release
- PAT token problem - user not found
- Ruleset problem